test/duct/logger/timbre_test.clj

```diff
@@ -21,13 +22,13 @@
                 ::timbre/println {}}
         logger (::logger/timbre (ig/init config))]
     (is (re-matches
-         #"(?x)\d\d-\d\d-\d\d\ \d\d:\d\d:\d\d\ [^\s]+\ 
```

Comment:
The whitespace at the end of this line is significant. Please put it back along with the others, otherwise the tests will fail.